### PR TITLE
fix: page goes white when clicking chip

### PR DESCRIPTION
**Pull request by alexandereneroth**
Used git bisect and found that the issue originated in index.css.

Rewrote it, it works but its not great.. Includes setting heights with JS. However there seems to be no other way to achieve our desired layout.

### CODE REVIEW

**Comment by mJarsater** [comment]
Love this ❤️

**Review by mJarsater — APPROVED** [review]

**Review by theJohnnyMe — APPROVED** [review]
Did clicked around and played with it, did not find anything broken.
Thanks for a big chuck of work Alex!

File: src/MainBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { TdsBanner, TdsLink } from '@scania/tegel-react';
+
+interface MainBannerProps {
+  style?: React.CSSProperties;
+  className?: string;
+  onClose?: () => void;
+}
+
+const MainBanner: React.FC<MainBannerProps> = ({ style, className, onClose }) => {
```
Comment (timrombergjakobsson) [inline src/MainBanner.tsx:10]: Just realised, that we usually been doing it like this with interfaces
```suggestion
const MainBanner = ({ style, className, onClose }: MainBannerProps) => {
```

Comment (alexandereneroth) [reply src/MainBanner.tsx:10]: Ah yes, I would say using React.FC is a more complete way of doing things though.

**Pull request merged**